README.md

```diff
@@ -1,7 +1,5 @@
 # Bare Metal Operator
 
-[![Ubuntu V1alpha2 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a2_integration_test_ubuntu/badge/icon?subject=Ubuntu%20E2E%20V1alpha2)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a2_integration_test_ubuntu)
-[![CentOS V1alpha2 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a2_integration_test_centos/badge/icon?subject=CentOS%20E2E%20V1alpha2)](https://jenkins.nordix.org/view/Airship/job/airship_master_v1a2_integration_test_centos)
```

Comment:
Is this change related?

Reply:
It is totally different thing, not related to the actual PR.  
It is just that we have dropped v1alpha2 CI now, so there will not be any CI status reported for v1alpha3 integration tests. 
So, I just wanted to fix it with the same PR as mentioned in the commit message above.

Reply:
Do you want to do it on a separate PR ?

Reply:
I have removed unrelated changes. Will open a separate PR for that.